site/src/pages/UsersPage/UsersTable/UserGroupsCell.tsx

```diff
@@ -31,18 +30,12 @@ export function UserGroupsCell({ userGroups }: GroupsCellProps) {
       ) : (
         <Popover mode="hover">
           <PopoverTrigger>
-            <Button
+            <button
               css={{
-                justifyContent: "flex-start",
-                fontSize: theme.typography.body2.fontSize,
-                lineHeight: theme.typography.body2.lineHeight,
-                fontWeight: 400,
+                cursor: "pointer",
+                backgroundColor: "transparent",
```

Comment:
Looks like this needs an explicit color now, too – the stories are showing that the text is now the same color as the background

https://www.chromatic.com/component?appId=624de63c6aacee003aa84340&csfId=pages-userspage-userstable&specName=Example&buildNumber=15193&k=66424a1ac22385f4368607db-1200px-interactive-true&h=7&b=-1

Reply:
Good catch!